common.mk

```diff
@@ -0,0 +1,5 @@
+SHELL=/bin/bash
+
+ifeq ($(findstring python3.6, $(shell ls -ls /usr/bin/python* | grep python3.6 2>&1)),)
```

Comment:
Do you need the trailing comma (next to last character)?

Reply:
Yes, that comma is needed as it seems part of the GNU `make` syntax. Throws an error without it.

Reply:
Not sure if this is a typical set up or not... Also added `which python`
```
~$ ls -ls /usr/bin/python*
32 -rwxr-xr-x  1 root  wheel  66576 Jul 14  2017 /usr/bin/python
 0 -rwxr-xr-x  5 root  wheel    925 Feb  7  2017 /usr/bin/python-config
 8 lrwxr-xr-x  1 root  wheel     75 Jul 15  2017 /usr/bin/python2.6 -> ../../System/Library/Frameworks/Python.framework/Versions/2.6/bin/python2.6
 8 lrwxr-xr-x  1 root  wheel     82 Jul 15  2017 /usr/bin/python2.6-config -> ../../System/Library/Frameworks/Python.framework/Versions/2.6/bin/python2.6-config
 8 lrwxr-xr-x  1 root  wheel     75 Jul 15  2017 /usr/bin/python2.7 -> ../../System/Library/Frameworks/Python.framework/Versions/2.7/bin/python2.7
 8 lrwxr-xr-x  1 root  wheel     82 Jul 15  2017 /usr/bin/python2.7-config -> ../../System/Library/Frameworks/Python.framework/Versions/2.7/bin/python2.7-config
32 -rwxr-xr-x  1 root  wheel  66576 Jul 14  2017 /usr/bin/pythonw
 8 lrwxr-xr-x  1 root  wheel     76 Jul 15  2017 /usr/bin/pythonw2.6 -> ../../System/Library/Frameworks/Python.framework/Versions/2.6/bin/pythonw2.6
 8 lrwxr-xr-x  1 root  wheel     76 Jul 15  2017 /usr/bin/pythonw2.7 -> ../../System/Library/Frameworks/Python.framework/Versions/2.7/bin/pythonw2.7
~$ which python
/anaconda3/bin/python
```